### ISSUE TITLE
Plan to take classes

### ISSUE THREAD

**Issue by leejk9592** [HyOsori/Osori-SelfDrivingWithGTA5#6]
# [Self Driving With GTA5](https://pythonprogramming.net/game-frames-open-cv-python-plays-gta-v/)

### ~ 10.21
- [x] Reading game frames in Python with OpenCV - Python Plays GTA V
- [x] OpenCV basics - Python Plays GTA V
- [x] Direct Input to Game - Python Plays GTA V
- [x] Region of Interest for finding lanes - Python Plays GTA V
- [x] Hough Lines - Python Plays GTA V
- [x] Finding Lanes for our self driving car - Python Plays GTA V
- [x] Self Driving Car - Python Plays GTA V

# [모두를 위한 ML/DL](https://hunkim.github.io/ml/)

###  ~ 10.28
- [x] Summary & Concept/Terms
- [x] Linear Regression
- [x] Minimize Cost Function Of Linear Regression
- [x] Linear Regression For Multiple Features
- [x] Logistic (Regression) Classification

### ~ 11.04
- [x] Softmax Regression (Multinomial Logistic Regression)
- [x] Actual Use Of Machine Learning And Several Tips
- [x] Basic Concepts Of Deep Learning & Problems/Solutions
- [x] XOR Problem And Learning Method & Back-Propagation
- [x] ReLU And Determine The Initial Values

### ~ 11.11
- [x] Convolutional Neural Networks
- [x] Recurrent Neural Network 

### ~ 11.18
- [ ] ~Lecture 1: Summary~
- [ ] ~Lecture 2: Open-AI GYM~
- [ ] ~Lecture 3: Dummy Q-learning (table)~
- [ ] ~Lecture 4: Q-learning exploit&exploration and discounted reward~

### ~ 11.25
- [ ] ~Lecture 5: Q-learning in non-deterministic world~
- [ ] ~Lecture 6: Q-Network~
- [ ] ~Lecture 7: DQN(Deep Q Network)~

**강화학습 강의는 프로젝트에도 쓰이지 않을 것 같아 나중에 개인적으로 들어볼 예정입니다**
~사실은 여행가서 노느라...~
# [Self Driving With GTA5 & ML](https://pythonprogramming.net/next-steps-python-plays-gta-v/)

### ~ 11.25
- [ ] Next steps for Deep Learning self driving car - Python Plays GTA V
- [ ] Training data for self driving car neural network- Python Plays GTA V
- [ ] Balancing neural network training data- Python Plays GTA V
- [ ] Training Self-Driving Car neural network- Python Plays GTA V
- [ ] Testing self-driving car neural network- Python Plays GTA V

### ~ 12.02
- [ ] A more interesting self-driving AI - Python Plays GTA V
- [ ] Object detection with Tensorflow - Self Driving Cars in GTA
- [ ] Determining other vehicle distances and collision warning - Self Driving Cars in GTA
- [ ] Getting the Agent a Vehicle- Python Plays GTA V
- [ ] Acquiring a Vehicle for the Agent - Python Plays GTA V
